src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmBrowserTestRunner.cs

```diff
@@ -213,9 +213,8 @@ private void RunSeleniumLogMessagePump(IWebDriver driver, CancellationToken toke
                         {
                             if (logEntry.Level == SeleniumLogLevel.Severe)
                             {
-                                // throw on driver errors, or other errors that show up
-                                // in the console log
-                                throw new ArgumentException(logEntry.Message);
+                                _logger.LogError($"[out of order message from the {logType}]: {logEntry.Message}");
```

Comment:
@radical would it be possible to add a comment here explaining the situation? I think it might be valuable in the future when we read this

Reply:
It's explained in the commit message, but I have added a comment with some info anyway.